syntax = "proto3";

//// https://protobuf.dev/reference/cpp/cpp-generated/#message
//option optimize_for = CODE_SIZE;

import "componentsfile.proto";
import "attrlistfile.proto";
import "featuresfile.proto";

package Odb.Lib.Protobuf;

//
//  LayerDirectory.h
//

message LayerDirectory  {

    optional string name = 1;
    optional string path = 2;

    optional ComponentsFile components = 3;
    optional AttrListFile attrlistFile = 4;
    optional FeaturesFile featureFile = 5;

} 